projects/Unit/Unit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
```

Comment:
Just noticed that this went from `netcoreapp3.1` to `netcoreapp2.1`.  If you only want to test one .NET Core, then the latest LTS release seems to be the one to choose, so I'd keep it at  `netcoreapp3.1`.

You could also do `net461;netcoreapp3.1;netcoreapp2.1`, but I'm not sure if that would be worth bothering with.

Reply:
I have considered switching our tutorials to `netcoreapp3.1` but then had to revert it for unrelated reasons. Targeting the latest sounds good to me.

Reply:
Thanks, that makes sense. We're going to test `net461` on Windows only because testing that framework on Linux requires Mono which really who is going to be using at this point?!?!